index.html

```diff
@@ -1,157 +1,174 @@
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul class="menu">
+        <li class="menu-items"><a href="#home">Hello</a></li>
+        <li class="menu-items"><a href="#projects">Portfolio</a></li>
+        <li class="menu-items"><a href="#about">About</a></li>
+        <li class="menu-items"><a href="#form">Contact</a></li>
+      </ul>
+      <button class="ham" type="button">
+        <span class="br-1"></span>
+        <span class="br-2"></span>
+        <span class="br-3"></span>
+      </button>
+    </nav>
+  </header>
+  <section class="home" id="home">
+    <div class="container">
+      <div class="desktophome">
+        <h1 class="logo"> Rizo<br> Abdunazarov</h1>
+        <p class="self">
+          I am a software developer - that is, a person who creates software
+          entirely. I help with the design or writing of the software, usually I
+          am the one who manages the project or just does the basic work. Like
+          most software developers, I am a leading software engineer.
+        </p>
+        <button class="btn" type="button">Start collaboration</button>
       </div>
-      <div>
-        <img
-          class="projectimages"
-          src="images/project2.png"
-          alt="project image"
-        />
-        <div class="projectcard">
-          <h3 class="projecttitle">Projects name goes here</h3>
-          <p class="languagebadge">
-            <span class="language">HTMLS/CSS</span>
-            <span class="language">Ruby on Rails</span>
-            <span class="language">JavaScript</span>
-          </p>
-          <br />
-          <button type="button" class="btn1">See this project</button>
-        </div>
+      <img class="homeimg" src="images/landingdesk.png" alt="landingdesk" />
+    </div>
+  </section>
+  <section class="projects" id="projects">
+    <h2 class="projects-title">Projects</h2>
+    <img src="images/project1.png" width="750" class="i1" alt="project1">
+    <div class="one projectcard">
+      <div class="thumbnail">
       </div>
-      <div>
-        <img
-          class="projectimages"
-          src="images/project3.png"
-          alt="project image"
-        />
-        <div class="projectcard">
-          <h3 class="projecttitle">Projects name goes here</h3>
-          <p class="languagebadge">
-            <span class="language">HTMLS/CSS</span>
-            <span class="language">Ruby on Rails</span>
-            <span class="language">JavaScript</span>
-          </p>
-          <br />
-          <button type="button" class="btn1">See this project</button>
-        </div>
+      <div class="project-summary">
+
+        <h2>Project's name goes here</h2>
+        <ul class="languagebadge">
+          <li class="language">HTML/CSS</li>
+          <li class="language">Ruby on Rails</li>
+          <li class="language">>Javascript</li>
+        </ul>
+        <button class="btn1" type="button">
+          See this project
+        </button>
```

Comment:
- [ ] (Required) The changes you made to the works section changed the design of your portfolio and shifted most things to the left. It would be nice if you fixed this and implement the changes suggested by the code reviewer for the _**works section**_.

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144730656-a2f8dc49-5c27-4c5f-874f-9aae1235ada9.png)

## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144730667-39ee3193-b31f-41c5-8968-1e7d65438c82.png)

### 👉  You will notice the white patch to the right for most sections of your portfolio for your design. Kindly work on it and make it responsive 🙏